_pages/governance.md

```diff
@@ -230,6 +230,18 @@ Members of the BIDS Contributors Group are encouraged to support the
 BIDS specification by supporting the members of the Maintainers Group in
 their responsibilities.
 
+An *Active Contributor* is defined as a contributor that is reachable via
+the email address they provided to the BIDS maintainers team,
+and responds to correspondence by the BIDS group
+(for example, maintainers/steering group or the BIDS community at large).
+Contributors may explicitly opt in or out of activity at any time.
+The maintainers group is responsible for providing transparent procedures for
+contributors to opt in/out as well as providing procedures for
+removing non-responsive members from the list of active contributors.
+In case of no response,
+a grace period of two years is in place before a formerly active contributor
+loses their "active" status.
```

Comment:
Note that I added a minor grammatical change from the adopted language: "is losing" -> "loses".

I do not believe this changes the intent or plain meaning of the sentence, but I can revert it if others disagree.